### PR TITLE
Cleanup and expose fast iterating serve targets

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: buildtools/webpack.gmfapps.js [hunk 1]
```diff
@@ -6,23 +6,22 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const plugins = [];
 const entry = {};
 
-for (const filename of ls('contribs/gmf/apps-webpack/*/index.html')) {
+const filenamePrefix = process.env.DEV_SERVER ? 'contribs/gmf/apps/' : '';
```
Comment (sbrunner) [inline buildtools/webpack.gmfapps.js]: Why do you need to prefix the path?

File: buildtools/webpack.gmfapps.js [hunk 2]
```diff
@@ -32,3 +31,11 @@ module.exports = {
   },
   plugins: plugins,
 };
+
+if (!process.env.DEV_SERVER) {
+  Object.assign(module.exports, {
+    output: {
+      path: path.resolve(__dirname, '../.build/contribs-gmf-apps'),
+    },
+  });
```
Comment (sbrunner) [inline buildtools/webpack.gmfapps.js]: This makes the dev server unworking?

File: package.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
   "description": "AngularJS OpenLayers Library",
   "scripts": {
     "prepublish": "make -f npm.mk install",
-    "serve-ngeo-examples": "NODE_ENV=dev TARGET=ngeo-examples webpack --watch & serve .build/examples-hosted/",
-    "serve-gmf-examples": "NODE_ENV=dev TARGET=gmf-examples webpack --watch & serve .build/examples-hosted/contribs/gmf/",
-    "serve-gmf-apps": "NODE_ENV=dev TARGET=gmf-apps webpack --watch & serve .build/contribs-gmf-apps/"
+    "serve-ngeo-examples": "DEV_SERVER=1 TARGET=ngeo-examples NODE_ENV=dev node_modules/.bin/webpack-dev-server --progress --watch --bail -d",
+    "serve-gmf-examples": "DEV_SERVER=1 TARGET=gmf-examples NODE_ENV=dev node_modules/.bin/webpack-dev-server --progress --watch --bail -d",
+    "serve-gmf-apps": "DEV_SERVER=1 TARGET=gmf-apps NODE_ENV=dev node_modules/.bin/webpack-dev-server --progress --watch --bail -d"
```
Comment (sbrunner) [inline package.json]: Can't we use webpack-dev-server directly?

Comment (gberaudo) [reply package.json]: Yes, you are right. Will simplify this.

**Comment by gberaudo** [comment]
Thanks for the review. I will remove https.js in the next PR.

**Pull request merged**